packages/lab/src/table/TD.tsx

```diff
@@ -6,10 +6,17 @@ import { withTableBaseName } from "./Table";
 
 import tableCss from "./Table.css";
 
-export interface TDProps extends ComponentPropsWithoutRef<"td"> {}
+export interface TDProps extends ComponentPropsWithoutRef<"td"> {
+  /**
+   * Specifies the alignment of the text within the `TD`.
+   *
+   * @default "left"
+   */
+  textAlign?: "left" | "right";
```

Comment:
we have props elsewhere called “align” with “left”, “right” and here we have “text-align”
which maps to CSS, but that uses”start”, “end”, so feels
likes the wires are crossed on our naming ?


Good one to discuss on our bi-weekly, how we approach naming.

Might either map

Reply:
This aligns with the Input textAlign prop https://github.com/jpmorganchase/salt-ds/blob/main/packages/core/src/input/Input.tsx#L57